frontend/webpage/src/components/Toolbar.vue

```diff
@@ -5,6 +5,7 @@
         </router-link>
         <ToolbarItem link="Home" iconClass="fa fa-home" />
         <ToolbarItem link="About" iconClass="fa fa-info" />
+        <ToolbarItem link="SearchRoom"/>
```

Comment:
To add a little magnification glass before the name
```suggestion
        <ToolbarItem link="SearchRoom" iconClass="fa fa-search"/>
```

Reply:
Fixed